tag-historian/nitaghistorian.yml

```diff
@@ -731,187 +680,129 @@ paths:
-        - in: body
-          name: Request body
-          required: true
-          description: Contains the identifiers of the historical values to delete.
+          description: OK
           schema:
-            description: Contains the identifiers of the historical values to delete.
             type: object
-            title: Tag Historian Delete Values By Id Request
+            title: Http Query Response Body
             properties:
-              ids:
-                description: The identifiers of the historical values to delete.
+              results:
+                description: An object containing the historical values for each requested tag that
+                  was successfully queried. The object's property names correspond to a tag path.
+                type: object
+                additionalProperties:
+                  $ref: '#/definitions/TypeAndValues'
+                example:
+                  system1.tag1:
+                    type: DOUBLE
+                    values:
+                      - value: '12.87'
+                        timestamp: '2019-03-13T09:51:38Z'
+                      - value: '3.2'
+                        timestamp: '2019-03-14T09:51:38Z'
+              errors:
+                $ref: '#/definitions/ErrorEntry'
+              failed:
+                description: The paths of the tags that had errors.
                 type: array
                 items:
                   type: string
-                  example: '5c88d2ae26b41e22f08b4407'
-            required: [ids]
-      responses:
-        204:
-          description: Success
-        401:
-          $ref: '#/responses/Unauthorized'
-        default:
-          $ref: '#/responses/Error'
-  /v1/tags/history/{path}/values/{id}:
-    get:
-      tags: [history]
-      summary: Read a historical value
-      description: Retrieves a single historical value and its metadata.
-      operationId: GetHistoricalValue
-      x-ni-operation: queryHistoricalTagValues
-      parameters:
-        - in: path
-          name: path
-          type: string
-          required: true
-          description: The path of the tag that owns the historical value being retrieved.
-          x-example: system1.tag1
-        - in: path
-          name: id
-          type: string
-          required: true
-          description: The identifier of the historical value to retrieve.
-          x-example: '5c88d2ae26b41e22f08b4407'
-      responses:
-        200:
-          description: Success
-          schema:
-            description: Contains the historical value, if found.
-            title: TagHistorianQueryResponseBody
-            type: object
-            properties:
-              tagHistorianValues:
-                $ref: '#/definitions/TagHistorianValues'
+                example: [system2.tag1]
         401:
           $ref: '#/responses/Unauthorized'
         default:
           $ref: '#/responses/Error'
-    put:
+  /v2/tags/download-csv:
```

Comment:
how about calling the route: /v2/tags/export-history
and a parameter "format" with "csv"

So this is a little more extendable.

Reply:
"responseFormat" is what we had before, and that felt more clear, so I went with that.